### DISCUSSION TITLE
[Urgent] MAUI some package causing  error NETSDK1082, but not sure how to find out which package is causing the issue

### DISCUSSION THREAD

**Discussion by KarthikHebbarMN** [dotnet/maui#14734] [Q&A]
Hey,

I'm building a MAUI project. I have project references. But when I build I get the following error 

>  error NETSDK1082: There was no runtime pack for Microsoft.AspNetCore.App available for the specified RuntimeIdentifier 'android-x64'

Just laying out the references to make things clearer. 
 The **MAUI** **project** has following references 
```
<ItemGroup>
	  <PackageReference Include="Microsoft.AppCenter.Analytics" Version="5.0.1" />
	  <PackageReference Include="Microsoft.AppCenter.Crashes" Version="5.0.1" />
	  <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
	  <PackageReference Include="Sharpnado.Tabs.Maui" Version="3.0.0" />
	</ItemGroup>

	<ItemGroup>
	  <ProjectReference Include="..\Models\Models.csproj" />
	  <ProjectReference Include="..\Utilities\Utilities.csproj" />
	</ItemGroup>
```

Side note : Both The Uitlities And Models project compiled for android, I tested by targeting them for android when building and they build successfully. 

**Utilities Package references** 
 
```
  <ItemGroup>
    <PackageReference Include="Magick.NET-Q16-AnyCPU" Version="8.2.1" />
    <PackageReference Include="MySql.Data" Version="8.0.32.1" />
    <PackageReference Include="Microsoft.Data.Sqlite" Version="7.0.4" />
    <PackageReference Include="NetVips" Version="2.0.1" />
    <PackageReference Include="NetVips.Native" Version="8.11.0" />
    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
    <PackageReference Include="StackExchange.Redis" Version="2.6.80" />
    <PackageReference Include="System.Data.SQLite.Core" Version="1.0.117" />
  </ItemGroup>

```
**Model Package References** 
```
  <ItemGroup>
    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
  </ItemGroup>
```
Now confusing part is all packages are compatible with dotnet 6 and from what I could scan through my projects we are not at all referring and Asp.Net libraries in code as well as in Depedency tree shown in Visual Studio. 
Still confused as to why this error is popping up ? 

We could remove a package at time and find out. But was looking if nuget had some commandline arguments to make this process less tedious. 
Dependency causing the issue **"Microsoft.AspNetCore.App"** 
Been blocked with this for a week now. Tried everything I was able to find on the net.

Any help is appreciated

**Comment by KarthikHebbarMN** [comment]
Any suggestions please ?

**Comment by Irene933** [comment]
Did you find a solution?
